modules/neo4j/slices_test.go

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [https://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
```

Comment:
@fbiville do you think we can remove this license header? All the library is distributed under MIT, and not sure if this subtle change for this small piece of code could cause confusion on the downstream users (not an expert in laws BTW).

I'd recommend removing it and keeping the repository's license file as the default one if possible.

Reply:
fixed in https://github.com/testcontainers/testcontainers-go/pull/992/commits/e58d928bbceeaf5929f469c74b961333903d1c22